modules/library/api/src/main/java/org/geotools/geometry/jts/ReferencedEnvelope.java

```diff
@@ -875,21 +897,7 @@ public static ReferencedEnvelope reference(Envelope e) {
             return new ReferencedEnvelope(e, null);
         }
     }
-
```

Comment:
that was the case for the methods considering a ReferencedEnvelope as a parameter (which is an OGC and a JTS envelope), but the bounding box is only an OGC envelope, so there was no problem.